### ISSUE TITLE
Speakup screen reader randomly stops speaking while configuring options

### ISSUE THREAD

**Issue by slannon97** [archlinux/archinstall#3748]
I'm totally blind and use the archinstall script to help install the system. This issue has been present for months. I'm using the ISO from August 1 and am still experiencing severe issues. Here are the steps to reproduce.

Boot the Arch Linux X64 ISO from August 1.
Immediately press down arrow and enter to start with speech output.
When Speakup begins talking, press Numpad * or I believe Caps Lock-Shift-8 if using a laptop without a numeric keypad to put Speakup into highlight tracking mode.
Type archinstall and press enter. Use the arrow keys and enter to make choices. Observe that speech randomly stops while selecting options and/or moving through lists. What causes this problem, and can it be fixed quickly? I can probably install Arch manually, but I prefer the archinstall script because it significantly simplifies the process.

**Comment by svartkanin** [comment]
We have migrated the UI to textual and hopefully screenreader support should be working better now
